tests/Functional/NewRelicCommandsTest.php

```diff
@@ -21,40 +21,38 @@ class NewRelicCommandsTest extends TestCase
      * @covers \Pantheon\Terminus\Commands\NewRelic\InfoCommand
      *
      * @group new-relic
-     * @group long_fixme
+     * @group long
      */
-    public function testNewRelicInfoEnableDisable()
+    public function testNewRelicInfoEnableDisableCommands()
```

Comment:
If we're changing the function names to include `Commands`, maybe we should make the same change to the other functions in `PlanCommandsTest.php`:

```
testPlanListCommand -> testPlanListCommands
testPlanInfoCommand -> testPlanInfoCommands
testSetPanCommand   -> testSetPlanCommands
```

Reply:
No, "s" at the end if a test covers multiple commands only

Reply:
Okay, understood.